### ISSUE TITLE
pdf generation in background thread

### ISSUE THREAD

**Issue by mahamedh** [dai-shi/react-hooks-worker#50]
is it possible to generate pdf using worker

**Comment by mahamedh** [comment]
> probably? what's the pdf generation library in your mind?

yes i am using jspdf library in react web development, i am leading some latency when generating pdf,to provide good user experinece for the user  i need to run the pdf generation in background....

**Comment by mahamedh** [comment]
Uncaught (in promise) ReferenceError: document is not defined 
i have tried jspdf pdf generation using react hooks worker i am facing above error 
@dai-shi

**Comment by dai-shi** [comment]
> document is not defined

Ah, this means the library doesn't support running in web workers.

**Comment by mahamedh** [comment]
@dai-shi  thanks for update save my time, any pdf gneration library developed on top web workers?

**Comment by dai-shi** [comment]
Hmm, no, I don't know.
https://www.npmjs.com/search?q=pdf%20generation
Maybe you want to find something that works both nodejs and browser.
